### DISCUSSION TITLE
Is io.appium.java_client.internal.ElementMap.getElementClass suppott removed from appium 8.1.1?

### DISCUSSION THREAD

**Discussion by gargpriya01** [appium/java-client#1731] [Q&A]
I am unable to use ElementMap after upgrading to appium java client 8.1.1. 

Does anyone know why this is so or an alternate solution to this?

**Comment by valfirst** [comment]
please check "Migration Guide": https://github.com/appium/java-client/blob/master/docs/v7-to-v8-migration-guide.md#mobileelement
